src/dotnet/Chat.Service/ChatsBackend.cs

```diff
@@ -123,6 +123,52 @@ public virtual async Task<ApiArray<ChatId>> GetPublicChatIdsFor(PlaceId placeId,
         return sChatIds.Select(x => new ChatId(x)).Where(x => !x.IsPlaceRootChat).ToApiArray();
     }
 
+    // [ComputeMethod]
+    public virtual async Task<ApiArray<ChatId>> GetPrivateChatIdsForUser(UserId userId, PlaceId? placeId, CancellationToken cancellationToken)
```

Comment:
we search for user chats via ContactsBackend, not directly through ChatsBackend.

Reply:
please take a look at already existing ContactsBackend.ListIdsForSearch, ...
https://github.com/Actual-Chat/actual-chat/blob/a1b841745c7e0983fd41badb1ded7ba2808f038f/src/dotnet/Contacts.Service/ContactsBackend.cs#L105